packages/node_modules/@ciscospark/plugin-feature/test/integration/spec/feature.js

```diff
@@ -71,9 +72,12 @@ describe(`plugin-feature`, function() {
     }];
 
     it(`sets a value for two user feature toggle`, () => {
+      spark.feature.setFeature(`user`, `key1`, false);
       return spark.feature.setBundledFeatures(featureUpdateArray)
         .then((res) => {
           assert.isDefined(res);
+          assert.equal(find(res.serialize().user, {key: `key1`}).value, `value1`);
+          assert.equal(find(res.serialize().user, {key: `key2`}).value, `value2`);
           assert.becomes(spark.feature.getFeature(`user`, `key1`), `value1`);
```

Comment:
These assert.becomes need to return to provide any values. Looks like it slipped through before, but let's get it fixed while you're looking at it. 

```js
return Promise.all([
  assert.becomes(spark.feature.getFeature(`user`, `key1`), `value1`),
  assert.becomes(spark.feature.getFeature(`user`, `key2`), `value2`)
]);
```